### ISSUE TITLE
Warning: Unknown props

### ISSUE THREAD

**Issue by msimonc** [Jishun/react-validated-input#4]
Now getting:

warning.js:44 Warning: Unknown props `validate`, `rules` on <input> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
    in input
    in span
    in div
    in ValidatedInput (created by _class)
    in div (created by _class)
    in div (created by _class)
    in div (created by _class)
    in _class (created by RouterContext)

**Comment by Jishun** [comment]
Hmm Mark

Are you able to get me a code snippet?

I'll try it tomorrow morning

Thanks!
On Tue, Jul 26, 2016 at 9:46 PM, Mark Simon <notifications@github.com<mailto:notifications@github.com>> wrote:

Now getting:

warning.js:44 Warning: Unknown props validate, rules on tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
in input
in span
in div
in ValidatedInput (created by _class)
in div (created by _class)
in div (created by _class)
in div (created by _class)
in _class (created by RouterContext)

**Comment by msimonc** [comment]
Nothing special:  (it might be my ES6 build environment again..) Thx!!

import Input, \* as inputHelper from 'react-validated-input'
...
                        <Input type="text" validate={this.state.validation} instance={this.state.valiData}
                               placeholder="Email"
                               propertyKey="email"
                               rules={{presence: true, length: {minimum: 2}}}></Input>

```
                    <Input type="password" validate={this.state.validation} instance={this.state.valiData}
                           placeholder="Password"
                           propertyKey="password"
                           rules={{presence: true, length: {minimum: 2}}}></Input>

                    <Button bsStyle='success' onClick={this.doValidate}>Login</Button>
```

...
    doValidate() {
        inputHelper.validate(this, [this.state.valiData]).then(() => {
                // console.log('no errors ');
                this.handleValidSubmit();
            },
            (errorMessages) => {
                console.log('errorMessages ', errorMessages);
                Util.showAlert('invalid');
            }
        ).catch(Util.catchError);
    }

**Comment by Jishun** [comment]
updated and published version 0.0.1
the issue was seen because react version 15 added more validations. removed the redundant props

**Comment by msimonc** [comment]
awesome; fixed!
